hudi-hadoop-mr/src/test/java/org/apache/hudi/hadoop/realtime/TestHoodieRealtimeRecordReader.java

```diff
@@ -155,7 +155,7 @@ public void testReader(boolean partitioned) throws Exception {
         } else {
           writer =
               InputFormatTestUtil.writeDataBlockToLogFile(partitionDir, fs, schema, "fileid0", baseInstant,
-                  instantTime, 100, 0, logVersion);
+                  instantTime, 120, 0, logVersion);
```

Comment:
can you explain why this was needed? This test was already adding inserts into logs? and now with this change we can actually read this.?

Reply:
I added 20 more records because the first 100 log records was expected to merge with the initial 100 parquet records, so after merge, we should read out 120 records actually.